charts/terraform-pod/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.11
+version: 0.0.12
```

Comment:
Please update also `appVersion` in this file to `1.5.3`

Reply:
Done